production/install

```diff
@@ -1285,7 +1285,25 @@ if [ "${ELEMENTS_LIQUID_ENABLE}" = ON ];then
         ;;
 
         Debian)
-            osSudo "${ROOT_USER}" install -c -o "${ROOT_USER}" -g "${ROOT_GROUP}" -m 644 "${MEMPOOL_HOME}/${MEMPOOL_REPO_NAME}/production/linux/liquid.service" "${DEBIAN_SERVICE_HOME}"
+            osSudo "${ROOT_USER}" install -c -o "${ROOT_USER}" -g "${ROOT_GROUP}" -m 644 "${MEMPOOL_HOME}/${MEMPOOL_REPO_NAME}/production/linux/elements.service" "${DEBIAN_SERVICE_HOME}"
+        ;;
+    esac
+fi
+
+#######################################
+# Bitcoin instance for Liquid Testnet #
+#######################################
+
+if [ "${ELEMENTS_LIQUID_ENABLE}" = ON ];then
+    echo "[*] Installing Bitcoin Liquid service"
+    case $OS in
+
+        FreeBSD)
+            echo "[*] FIXME: Bitcoin Liquid service must be installed manually on FreeBSD"
+        ;;
+
+        Debian)
+            osSudo "${ROOT_USER}" install -c -o "${ROOT_USER}" -g "${ROOT_GROUP}" -m 644 "${MEMPOOL_HOME}/${MEMPOOL_REPO_NAME}/production/linux/elements-testnet.service" "${DEBIAN_SERVICE_HOME}"
```

Comment:
I think we should try to be consistent with the network names:

- Bitcoin mainnet
- Bitcoin testnet
- Bitcoin signet
- Elements liquid
- Elements liquidtestnet